cls/SourceControl/Git/Extension.cls

```diff
@@ -248,6 +248,18 @@ Method IsInSourceControl(InternalName As %String) As %Boolean [ CodeMode = expre
 InternalName'="" && ##class(Utils).IsInSourceControl(##class(Utils).NormalizeInternalName(InternalName))
 }
 
+/// Called before an item is deleted.
+Method OnBeforeDelete(InternalName As %String) As %Status
+{
+    set context = ##class(SourceControl.Git.PackageManagerContext).ForInternalName(InternalName)
+    set InternalName = ##class(Utils).NormalizeInternalName(InternalName)
```

Comment:
Nit: full classnames are preferred (though other code in the same neighborhood might not be great about that)

Reply:
This happens in multiple places all throughout the project. It might be better to create a separate PR for finding and refactoring all these instances.